Nodejs/Product/Nodejs/Project/NodejsProjectNode.cs

```diff
@@ -577,9 +577,9 @@ public NodejsProjectNode(NodejsProjectPackage package)
                 return false;
             }
 
-            var relativeFile = CommonUtils.GetRelativeFilePath(this.FullPathToChildren, fileNode.Url);
+            var relativeFile = CommonUtils.GetRelativeFilePath(this.FullPathToChildren, fileNode.Url);
             if (this._analyzer != null && this._analyzer.Project != null 
-                && this._analyzer.Project.Limits.IsPathExceedNestingLimit(relativeFile)) {
+                && this._analyzer.Project.Limits.IsPathExceedNestingLimit(relativeFile)) {
```

Comment:
it looks like there is an extra space after the null - let me try to delete it and see if that does the trick.